test/modules/core/lib/deck.spec.js

```diff
@@ -167,3 +168,75 @@ test('Deck#auto view state', t => {
     }
   });
 });
+
+test('Deck#dataManager', async t => {
+  const layer1 = new ScatterplotLayer({
+    id: 'scatterplot-global-data',
+    data: 'deck://pins',
+    getPosition: d => d.position
+  });
+  const layer2 = new ScatterplotLayer({
+    id: 'scatterplot-shared-data-A',
+    data: 'cities.json',
+    getPosition: d => d.position
+  });
+  const layer3 = new ScatterplotLayer({
+    id: 'scatterplot-shared-data-B',
+    data: 'cities.json',
+    getPosition: d => d.position
+  });
+
+  const deck = new Deck({
+    gl,
+    width: 1,
+    height: 1,
+    // This is required because the jsdom canvas does not have client width/height
+    autoResizeDrawingBuffer: gl.canvas.clientWidth > 0,
+
+    viewState: {
+      longitude: 0,
+      latitude: 0,
+      zoom: 0
+    },
+
```

Comment:
Are we going to have a `Deck.resources` prop so we can specify these declaratively.

Reply:
Yes eventually